app2.js

```diff
@@ -0,0 +1,45 @@
+require('dotenv').config();
+
+const express = require('express');
+const mongoose = require('mongoose');
+
+const User = require('./models/users');
+
+const mongoUrl = `${process.env.MONGODB_URI}${process.env.DB_NAME}`;
+const port = process.env.PORT;
+
+mongoose.connect(mongoUrl);
+
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.get('/users', async(req, res) => {
+	const users = await User.find({});
+	res.send(users);
+});
+
+app.post('/users', async(req, res) => {
+	try {
+		const user = await User.create(req.body);
```

Comment:
Dont flood routes by business logic. Add controllers to handle requests and services for business logic